applications/TrilinosApplication/custom_python/add_trilinos_space_to_python.cpp

```diff
@@ -287,7 +300,10 @@ void  AddBasicOperations(pybind11::module& m)
     .def("CreateEmptyMatrixPointer", CreateEmptyMatrixPointer)
     .def("CreateEmptyVectorPointer", CreateEmptyVectorPointer)
     .def("ReadMatrixMarketMatrix", ReadMatrixMarketMatrix)
+    .def("ReadMatrixMarketVector", ReadMatrixMarketVector)
     .def("SetValue", SetValue)
+    //.def("GetValue", GetValue) //deliberately commented out. Only works for local Ids
```

Comment:
could we implement a check for the Id if it is local to not lose this functionality?

Reply:
i did not remove this, it was simply not exported. Note that the input is a global id, and if it is not local an error is thrown, so i would not export this... instead you should use the GatherValues function which i now verified to work